### ISSUE TITLE
speech dictionary specific to the synthesizer (all its voices)

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#212]
**Reported by aleksey_s on 2008-10-27 14:00**
for newfon synthesizer (which has 4 voices) i need to make some text preprocessing. currently it is done with regexprs inside the module, but many of them (and many, which will be added later) might be expanded to the nvda speech dictionaries mechanizm. if developers not opossite to this i can write patch myself.
**Blocked by #219**

**Comment by leonardder** [comment]
Superseded by #219
